yaml_decode/readme.md

```diff
@@ -0,0 +1 @@
+[![Pub Package](https://img.shields.io/pub/v/yaml_decode.svg)](https://pub.dartlang.org/packages/yaml_decode)
```

Comment:
A real readme would be good if you can, also consider travis build badges etc

Reply:
will follow-up with a real readme, tests, example, etc